### PR TITLE
New zingolib version & openssl 3.1.3 updated & create build_andoid 007 docker image

### PR DESCRIPTION
**Pull request by juanky201271**
- android ndk r26b Latest.
- cargo-ndk removed because we are not using it.
- openssl updated to the Latest version 3.1.3.
- new build_android docker image 007.
- new zingolib version.
- new test for `exportufvk`.

### CODE REVIEW

File: rust/Dockerfile [hunk 1]
```diff
@@ -11,21 +11,21 @@ COPY ios/ /opt/zingo/rust/ios/
 COPY zingomobile_utils/ /opt/zingo/rust/zingomobile_utils/
 
 RUN AR=llvm-ar LD=ld RANLIB=llvm-ranlib CC=aarch64-linux-android24-clang \
-    OPENSSL_DIR=/opt/openssl-3.1.2/aarch64 cargo +nightly build -Z build-std \
+    OPENSSL_DIR=/opt/openssl-3.1.3/aarch64 cargo +nightly build -Z build-std \
```
Comment (zancas) [inline rust/Dockerfile:14]: Do we still need `+nightly`?

**Review by zancas — COMMENTED** [review]
UtAck

**Comment by zancas** [comment]
Probably @Oscar-Pepper should be the reviewer on this.

File: rust/android/builder/Dockerfile [hunk 2]
```diff
@@ -46,7 +46,7 @@ RUN rustup target add \
     armv7-linux-androideabi \
     i686-linux-android \
     x86_64-linux-android
-RUN cargo install cargo-ndk
+# RUN cargo install cargo-ndk
```
Comment (Oscar-Pepper) [inline rust/android/builder/Dockerfile]: if this is obselete shall we remove this line?

File: rust/Dockerfile (hunk 1 above)
Comment (Oscar-Pepper) [reply rust/Dockerfile:14]: we need nightly for `build-std` (https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#build-std)
what this does and whether we need it is not something im too familiar with but a skim-read leads me to believe that this is to do with compiling std libs in the build?

**Review by Oscar-Pepper — DISMISSED** [review]

File: rust/android/builder/Dockerfile (hunk 2 above)
Comment (juanky201271) [reply rust/android/builder/Dockerfile]: sure... I will.

Comment (juanky201271) [reply rust/android/builder/Dockerfile]: done!

File: rust/Dockerfile (hunk 1 above)
Comment (juanky201271) [reply rust/Dockerfile:14]: Any news about this?

**Review by zancas — COMMENTED** [review]
LGTM, but I think @Oscar-Pepper needs to review.

**Review by Oscar-Pepper — APPROVED** [review]

**Pull request merged**